.github/workflows/ci-pipeline.yml

```diff
@@ -83,17 +89,134 @@ jobs:
             } else {
               core.warning(message);
             }
+  update-pr:
+    runs-on: ubuntu-latest
+    needs: [get-chart]
+    name: Automatically update README and CRDs
+    permissions:
+      contents: read
```

Comment:
`contents: write` will be required if `github.actor == 'bitnami-bot'`

Reply:
Modified permissions to `contents: write`, same as the readme-generator workflow 👍